apps/docs/pages/docs/api-reference/configuration/component-config.mdx

```diff
@@ -56,18 +56,37 @@ const config = {
 
 #### Render props
 
-| Arg                                          | Example       | Type     |
-| -------------------------------------------- | ------------- | -------- |
-| [`id`](#id)                                  | `button-1234` | String   |
-| [`puck.dragRef`](#puckdragref)               | `null`        | Function |
-| [`puck.isEditing`](#puckisediting)           | `false`       | Boolean  |
-| [`puck.renderDropZone`](#puckrenderdropzone) | `() => {}`    | Function |
-| [`...props`](#props)                         | `{}`          | Object   |
+| Arg                                             | Example       | Type     |
+| ----------------------------------------------- | ------------- | -------- |
+| [`id`](#id)                                     | `button-1234` | String   |
+| [`puck.metadata`](/docs/api-reference/metadata) | `{}`          | Object   |
+| [`puck.dragRef`](#puckdragref)                  | `null`        | Function |
+| [`puck.isEditing`](#puckisediting)              | `false`       | Boolean  |
+| [`puck.renderDropZone`](#puckrenderdropzone)    | `() => {}`    | Function |
+| [`...props`](#props)                            | `{}`          | Object   |
 
 ##### `id`
 
 A unique ID generated by Puck for this component. You can optionally apply this, or use your own ID.
 
+##### `puck.metadata`
+
+Global metadata passed in to the `<Puck>` component. Can be used along with component props for rendering.
+Using metadata within the `render` API is non-destructive and will conserve the values saved in the component props.
```

Comment:
Think this can be simplified

```suggestion
Global metadata that is be passed in via the Puck [`metadata` prop](/docs/api-reference/components/puck#metadata).
```